certified-connectors/Revizto (Ireland)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1413 @@
+                  "default": 0,
+                  "type": "integer"
+                }
+              },
+              "required": [
+                "result",
+                "data",
+                "message"
+              ],
+              "type": "object"
+            }
+          }
+        }
+      }
+    },
+    "/v5/user/licenses": {
+      "get": {
+        "operationId": "Get__v5_user_get_licenses",
+        "tags": [
+          "Licenses"
+        ],
+        "summary": "Get current user's licenses",
+        "description": "Gets the licenses available to the current user.",
+        "x-ms-summary": "Get a list of all avaliable current user's lincenses",
+        "parameters": [],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "You are authorized, and the server returned a response",
+            "x-ms-summary": "You are authorized, and the server returned a response",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "description": "A JSON object with detailed response data",
+                  "x-ms-summary": "A JSON object with detailed response data",
+                  "properties": {
+                    "count": {
+                      "description": "Count of user licenses",
+                      "x-ms-summary": "Count of user licenses",
+                      "example": 1,
+                      "type": "integer"
+                    },
+                    "entities": {
+                      "description": "A JSON array that describes one or several licenses",
+                      "x-ms-summary": "A JSON array that describes one or several licenses",
+                      "items": {
+                        "$ref": "#/definitions/LicenseItem"
+                      },
+                      "type": "array"
+                    }
+                  },
+                  "required": [
+                    "entities",
+                    "count"
+                  ],
+                  "type": "object",
+                  "x-nullable": true
+                },
+                "message": {
+                  "description": "Response status",
+                  "x-ms-summary": "Response status",
+                  "default": "Success.",
+                  "title": "Status",
+                  "type": "string"
+                },
+                "result": {
+                  "description": "Response status code",
+                  "x-ms-summary": "Response status code",
+                  "title": "Code",
+                  "default": 0,
+                  "type": "integer"
+                }
+              },
+              "required": [
+                "result",
+                "data",
+                "message"
+              ]
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LicenseItem": {
```

Comment:
The required property 'type' is not defined in the object schema. ValuePath: /definitions/LicenseItem/required